frontend/src/components/ResetModal/ResetModal.test.tsx

```diff
@@ -31,20 +31,20 @@ describe('ResetModal', () => {
   });
 
   it('should render', () => {
-    renderWithProviders(<ResetModal />);
+    renderWithProviders(<ResetModal open={false} />);
```

Comment:
probs can make the `open` prop optional so we dont have to be forced to set it to `false` and instead just don't include the prop at all

Reply:
sure, my bad!